content-services/content-sources-backend
# Fixes 4267,4269: add user to template

## Summary
Adds created_by and last_updated_by fields to template, which is the username if the user that is modifying the template

Adds created_at and updated_at fields to template response

## Testing steps
1. To test through the API, you need an identity header the the username. You can use `./scripts/headers.sh <org_id> [username]`. By default the username is "snapUser".
2. Testing through the UI will already have the username in the header
3. Creating a template should set the created_by and last_updated_by fields
4. Updating a template should set the last_updated_by field
5. Also check for created_at and updated_at in template GET requests
 

## Checklist
